back/spec/lib/participation_method/survey_spec.rb

```diff
@@ -52,6 +52,12 @@
     end
   end
 
+  describe '#sign_in_required_for_posting?' do
+    it 'returns true' do
+      expect(participation_method.sign_in_required_for_posting?).to be true
```

Comment:
Is this correct? I can set up a project for an external survey with access rights to `Anyone`. Then an unregistered user can take that external survey.